Driverless AI/image-processing-in-driverless-ai/image-processing-in-driverless-ai.md

```diff
@@ -0,0 +1,595 @@
+
+To showcase how fine-tuning was enabled for the first approach to image processing in DAI, observe the steps you can take to rerun the experiment with fine-tuning: 
+
+In the **Experiments** section:
+
+1. Click the **three vertical dots** (located on the right side of the experiment) of the following experiment: `Embeddings-Transformer-Without-Fine-Tuning`
+
+![exp2-new-experiment](assets/exp2-new-experiment.png)
+
+2. Click the following option: **NEW EXPERIMENT WITH SAME SETTINGS**
+
+3. Rename the experiment to `Embeddings-Transformer-With-Fine-Tuning`
+
+4. Under the **IMAGE** tab located in the **EXPERT SETTINGS** click the **DISABLED** button under the following setting: **Enable Fine-tuning of pre-trained models used for image Transformer** 
+
+    - This setting will change from **DISABLED** to **ENABLED** 
+
+![enabled-fine-tuning](assets/enabled-fine-tuning.png)
+
+5. **LAUNCH EXPERIMENT**
+
+![exp2-launch-experiment](assets/exp2-launch-experiment.png) 
+
+When fine-tuning is enable, Driverless AI provides a list of possible image augmentations to apply while fine-tuning the **ImageNet pre-trained models** used for the **Image Transformer**. By default, **HorizontalFlip** is enabled, and for purposes of this tutorial, it was not changed.. This default setting can be found and change in the **IMAGE** tab inside the **EXPERT SETTINGS**. Please refer to the Driverless AI documentation right [here](https://docs.h2o.ai/driverless-ai/latest-stable/docs/userguide/expert_settings/image_settings.html#list-of-augmentations-for-fine-tuning-used-for-the-image-transformer) for a full list of all other augmentations. 
+
+![list-of-augmentations-for-fine-tuning-used-for-image-transformer](assets/list-of-augmentations-for-fine-tuning-used-for-image-transformer.png)
+
+**NOTE**: Augmentations for fine-tuning used for the Image Transformer is only available when Fine-tuning is **enabled**.  
+
+As well, when fine-tuning is enabled, you can specify the number of epochs for fine-tuning **ImageNet pre-trained** models used for the **Image Transformer**. This value defaults to **2**. This default setting can be found and change in the **IMAGE** tab inside the **EXPERT SETTINGS**.
+
+Now that you know how to rerun the experiment with fine-tuning let's explore the new experiment (Embeddings-Transformer-With-Fine-Tuning). 
+
+![finish-experiment-car_deals_with_fine_tuning](assets/finish-experiment-car_deals_with_fine_tuning.png)
+
+For our second experiment, we see that the validation score for the final pipeline is RMSE = 4114.0424 +/- 231.9882. Recall that the RMSE for the first experiment was 4058.833 +/- 154.7854. Enabling fine-tuning didn't improve the RMSE; instead, it did the opposite. The RMSE increase by ~55.209.
```

Comment:
Maybe the first experiment also hasn't used images in the final model. So, this comparison and conclusions afterwards could not be applied here.
I would rather change settings/dataset/etc. in order to show that Fine-Tuning helps improve the model quality.